.github/ISSUE_TEMPLATE/bug-issue.yml

```
@@ -0,0 +1,72 @@
+name: Bug report
+description: Report a very clearly broken issue.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # osu! bug report
+
+        Important to note that your issue may have already been reported before. Please check:
+        - Pinned issues, at the top of https://github.com/ppy/osu/issues.
+        - Current open `priority:0` issues, filterable [here](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority%3A0).
+        - And most importantly, search for your issue. If you find that it already exists, respond with a reaction or add any further information that may be helpful.
+
+  - type: dropdown
+    attributes:
+      label: Type
+      options:
+        - Crash to desktop
+        - Game behaviour
+        - Performance
+        - Cosmetic
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: What are the steps for reproducing the bug? Any additional details that might be of help?
```

just going to suggest to simplify the language by a lot

```suggestion
      description: How did you find the bug? Any additional details that might help?
```

---

I'm going to ask for @peppy's opinion on whether to simplify or keep as-is, since [one of the descriptions](https://github.com/ppy/osu/pull/17322#discussion_r829831031) has also been reworded to become as it is now.